packages/react/src/components/UIShell/__tests__/__snapshots__/HeaderGlobalAction-test.js.snap

```diff
@@ -36,7 +36,7 @@ exports[`HeaderGlobalAction should render 1`] = `
           xmlns="http://www.w3.org/2000/svg"
         >
           <path
```

Comment:
intentional?

Reply:
Unsure where that came from ... I know that jest failed the snapshot tests, so I did run an update ... I'll pull and see what happens

Reply:
So yeah, I'm pretty sure that's a side effect of running a `jest -u` - the tests were failing and that seems to sort it out ... let me know if that's not the right approach!